src/main/java/no/fdk/dataservicecatalog/service/DataServiceService.java

```diff
@@ -69,27 +60,27 @@ private DataService applyPatch(DataService dataService, Map<String, Object> part
                 .doOnError(error -> log.error("dataservice with id {} failed mapping {}", dataServiceId, error.getMessage()));
         return dataServiceMono.flatMap(dataService -> {
             dataService.setId(dataServiceId);
-            dataService.setCatalogId(catalogId);
+            dataService.setOrganizationId(catalogId);
```

Comment:
rename catalogId to organizationId

Reply:
catalogId means it comes from the url path